daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -971,6 +971,15 @@ message Expr {
     Expr interface_expr = 3;
   }
 
+  // Obtain an interface view
+  // *Available in versions >= 1.dev*
+  message ViewInterface {
+    TypeConName interface = 1;
+    TypeConName template = 2;
+    Type viewtype = 3;
+    Expr expr = 4;
+  }
```

Comment:
I do not really understand what we are trying to archive here. 
In my mind, a view is just a glorified method, so I do not expect we need more info that for `CallInterface`, that is to say I would expect something like:

```suggestion
  message ViewInterface {
    TypeConName interface = 1;
    Expr expr = 2;
  }
```
In particular I do not need the type since I can infer it from the definition of the interface, and in general I do not know the template ID.

Reply:
You're right - I've modified it to reflect CallInterface more closely